src/helpers/proofs/generateEmail.ts

```diff
@@ -0,0 +1,36 @@
+import { BadgeSourceType } from 'data'
+import { DataKey } from 'models/DataKey'
+import { ProofStore } from 'stores/ProofStore'
+import { getEddsaPublicKey } from 'helpers/proofs/attestor'
+import { handleError } from '@big-whale-labs/frontend-utils'
+import buildEmailProof from 'helpers/proofs/buildEmailProof'
+import checkNavigator from 'helpers/proofs/checkNavigator'
+
+export default async function generateEmail(
+  store: ProofStore,
+  original: string,
+  options: {
+    secret?: string
```

Comment:
Why don't we put `secret` outside of `options`?

Reply:
Because network and secret are not needed everywhere